### ISSUE TITLE
Long Boot Time (30s) on ComfyUI Load

### ISSUE THREAD

**Issue by franckdsf** [spacepxl/ComfyUI-Image-Filters#8]
Hi there :)

I'm using the "Restore Details" node from your ComfyUI-Image-Filters custom node in my serverless workflow. I've noticed that ComfyUI takes around 30 seconds to load the library when starting, significantly delaying the startup time.

**Questions:**
- Is this long boot time expected due to the full library loading?
- Are there any known performance bottlenecks in this process?
- Would it be possible to load only the "Restore Details" node (or other specific nodes) to avoid loading the whole library? If so, could you provide guidance on how to achieve that?

Any insights would be appreciated. Thank you for the great work!

**Comment by spacepxl** [comment]
Usually if a custom node pack takes a long time to load during startup, it's because it's the last one to load, and comfyui just dumps a bunch of other unrelated stuff into the loading time, making it look like it took ages to load when really it was more like 0.5s.

**Comment by mesbahmilad** [comment]
That is not true thought, I use this node pack in a number of workflows and it is the only node pack in any my workflows with a load time well over 20s. Never seen another node pack take more than 1s

**Comment by robertvoy** [comment]
This happens becuse of numba JIT compilation that is required by pymatting.
